IBPSA/Electrical/BaseClasses/PV/PVThermalEmpMountCloseToGround.mo

```diff
@@ -0,0 +1,36 @@
+within IBPSA.Electrical.BaseClasses.PV;
+model PVThermalEmpMountCloseToGround
+  "Empirical thermal model for PV cell with back close to ground (0 deg < til < 10 deg)"
+extends IBPSA.Electrical.BaseClasses.PV.BaseClasses.PartialPVThermalEmp;
+
+equation
+
+ TCel = HGloTil*(exp(-2.98-0.0471*winVel))+(TDryBul)+HGloTil/1000;
+
+  annotation (Icon(coordinateSystem(preserveAspectRatio=false)), Diagram(
+        coordinateSystem(preserveAspectRatio=false)),
+    Documentation(info="<html>
+<p>
+  Model for determining the cell temperature of a PV module mounted on
+  an open rack under operating conditions and under consideration of
+  the wind velocity.
+</p>
+<p>
+  <br/>
+</p>
+<h4>
+  <span style=\"color: #008000\">References</span>
+</h4>
```

Comment:
Don't format html, a tool should do this (and they use their own formatting).

Reply:
done